src/wallet/wallet.cpp

```diff
@@ -1835,21 +1835,14 @@ CAmount CWallet::GetDebit(const CTxIn &txin, const isminefilter& filter) const
 {
     {
         LOCK(cs_wallet);
-#if 0
```

Comment:
Consider restoring previous implementation that might be slightly more efficient even if it was less readable

Reply:
Agreed & done